doc/glossary.adoc

```diff
@@ -2,47 +2,58 @@
 
 == LivingDoc 2 - Core Domain
 
-Decision Table:: [[decision-table]] A table where each line represents
-a <<test-case,Test Case>>.  A column can contain either a <<test-input,Test
-Input>> or an <<expectation,Expectation>>.
+Decision Table:: [[decision-table]]
+A table where each row represents a <<test-case,Test Case>>.  A column can
+contain either a <<test-input,Test Input>> or an <<expectation,Expectation>>.
 
-Example:: A description of the behavior of the <<system-under-test,System under
+Example:: [[example]]
+A description of the behavior of the <<system-under-test,System under
 Test>>, which can be automatically verified by LivingDoc. <<scenario,Scenario>>
 and <<decision-table,Decision Table>> are examples.
 
-Executable Document:: [[executable-document]]A document describing the
-<<system-under-test,System under Test>>. It contains zero or more
-<<example,Examples>>.
-
-Expectation:: [[expectation]]A value that is passed to the <<fixture,Fixture>>
-to determine the outcome of a <<test-case,Test Case>>.
-
-Fixture:: [[fixture]]Coded by a developer, a fixture is used by an
-<<interpreter,Interpreter>> to execute a <<test-case,Test Case>>. The fixture
-maps <<test-input,Test Inputs>> to a series of interactions with the
-<<system-under-test,System under Test>> and evaluates
-<<expectations,Expectations>>.
-
-Interpreter:: [[interpreter]] Takes an <<example,Example>>, determines required
-<<fixture,Fixtures>> and executes specified <<test-case,Test Cases>>.
-
-Scenario:: [[scenario]]A table describing a <<test-case,Test Case>>. Each line
-of the table represents an individual test step, which consists of keywords,
-<<fixture-command,Fixture Commands>>, <<test-input,Test Inputs>> and
+Executable Document:: [[executable-document]]
+A document describing the <<system-under-test,System under Test>>. It contains
+zero or more <<example,Examples>>.
+
+Expectation:: [[expectation]]
+A value that is passed to the <<fixture,Fixture>> to determine the outcome of a
+<<test-case,Test Case>>.
+
+Fixture:: [[fixture]]
+Coded by a developer, a fixture is used by an <<interpreter,Interpreter>> to
+execute a <<test-case,Test Case>>. The fixture maps <<test-input,Test Inputs>>
+to a series of interactions with the <<system-under-test,System under Test>>
+and evaluates <<expectations,Expectations>>.
+
+Interpreter:: [[interpreter]]
+Takes an <<example,Example>>, determines required <<fixture,Fixtures>> and
+executes specified <<test-case,Test Cases>>.
+
+Scenario:: [[scenario]]
+An <<example, Example>> describing a single <<test-case,Test Case>> as a
+sequence of <<scenario-step, Steps>>.
+
+Scenario Step:: [[scenario-step]]
```

Comment:
Scenario Step: A single step of a _Scenario_ represented as a sentence containing either one or more _Test Inputs_ or one or more _Expectations_.